src/neo-vm/ExecutionEngine.cs

```diff
@@ -53,28 +58,23 @@ public void Execute()
 
         private void ExecuteOp(OpCode opcode, ExecutionContext context)
         {
-            if (opcode > OpCode.PUSH16 && opcode != OpCode.RET && context.PushOnly)
```

Comment:
Is it possible that this might effect reprocessing the chain? Seems like executions of `PushOnly = true` scripts might now succeed where before they failed.

Reply:
No. `PushOnly` will only be used in the verification process, and the verification result of history transactions will not change.